src/lib/comms/sol-lwm2m-bs-server.c

```diff
@@ -482,10 +492,15 @@ sol_lwm2m_bootstrap_server_delete_object_instance(struct sol_lwm2m_bootstrap_ser
     enum sol_coap_response_code response_code),
     const void *data)
 {
+    enum sol_lwm2m_path_props props;
+
     SOL_NULL_CHECK(server, -EINVAL);
     SOL_NULL_CHECK(client, -EINVAL);
     SOL_NULL_CHECK(path, -EINVAL);
 
+    props = sol_lwm2m_common_get_path_props(path);
+    SOL_EXP_CHECK(props != PATH_HAS_INSTANCE, -EINVAL);
```

Comment:
Indeed I forget that during bootstrap I can send a delete '/'. I will rename the enum value.